packages/sommelier.rb

```diff
@@ -96,14 +96,33 @@ def self.build
           WAYLAND_DISPLAY=wayland-0
           XDG_RUNTIME_DIR=/var/run/chrome
 
-          case "$(uname -m)" in
-          x86_64|i686)
-            sort_result="$(sort -V <<< "$(uname -r)"$'\\n'"4.16.0" | tail -n1)"
-            if [[ "${sort_result}" == "4.16.0" ]]; then
-              MESA_LOADER_DRIVER_OVERRIDE=i965
-            fi
-          ;;
-          esac
+          if grep -q GenuineIntel /proc/cpuinfo ;then
+            check_linux_version() {
+                maj_ver=$1
+                min_ver=$2
+                version_good=$(uname -r | awk 'BEGIN{ FS="."};
+                { if ($1 < '"${maj_ver}"') { print "N"; }
+                  else if ($1 == '"${maj_ver}"') {
+                      if ($2 < '"${min_ver}"') { print "N"; }
+                      else { print "Y"; }
+                  }
+                  else { print "Y"; }
+                }')
+
+                if [ "$version_good" = "N" ]; then
+                    return 1
+                fi
+            }
+            case "$(uname -m)" in
```

Comment:
Could this be `case ARCH in`?

Reply:
There is no ARCH env variable we export... And it doesn't make sense to get it by running `crew const ARCH`.

Reply:
Gotcha.